snapcraft/cli/_errors.py

```diff
@@ -143,14 +147,33 @@ def _handle_sentry_submission(exc_info) -> None:
             click.echo(_MSG_SEND_TO_SENTRY_THANKS)
 
 
-def _print_exception_message(exc_info, debug) -> None:
-    # Print exception in exc_info[1].
-    message = str(exc_info[1])
-    echo.error(message)
+def _print_snapcraft_exception_message(exception: errors.SnapcraftException):
+    parts = [exception.get_brief()]
+
+    resolution = exception.get_resolution()
+    if resolution:
+        parts.extend(["", "Recommended resolution:", resolution])
```

Comment:
why is the first element here ""?

Reply:
To add a newline.  It could be "\nRecommended..." as well

Reply:
We can leave it like this for now